clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -252,16 +254,17 @@ const PrivacyRequestForm: React.FC<PrivacyRequestFormProps> = ({
                 id="email"
                 isInvalid={touched.email && Boolean(errors.email)}
               >
+                <FormLabel>
+                  {action.identity_inputs.email === "required"
+                    ? "Email*"
+                    : "Email"}
+                </FormLabel>
                 <Input
                   id="email"
                   name="email"
                   type="email"
                   focusBorderColor="primary.500"
-                  placeholder={
-                    action.identity_inputs.email === "required"
-                      ? "Email*"
-                      : "Email"
-                  }
+                  placeholder="test-email@testemailaddress.com"
```

Comment:
`@example.com` is usually a better choice because https://example.com/ is officially reserved as a placeholder domain. [testemailaddress.com](https://testemailaddress.com) is an actual company's site.